src/components/Nav.jsx

```diff
@@ -0,0 +1,16 @@
+import Header from './Header';
+import TopicSelector from './TopicSelector';
+import SortBySelector from './SortBySelector';
+import UserCard from './UserCard';
+
+export default function Nav({ setTopic, setSortBy }) {
+    
+    return (
+        <nav>
+        <Header />
+        <TopicSelector />
+        <SortBySelector />
+        <UserCard />
```

Comment:
Some of these components are not yet being used, so they shouldn't really have been committed in this push. It's great that you're planning ahead, but always think when you're pushing if the component is necessary for what the ticket has asked. It can also be a common way of causing errors if your app is bloated with components that aren't needed yet. As Rich says "This is how we trip!" 😄